src/PROPOSAL/detail/PROPOSAL/scattering/stochastic_deflection/bremsstrahlung/BremsGinneken.cxx

```diff
@@ -40,12 +39,16 @@ double stochastic_deflection::BremsGinneken::get_rms_theta(double e_i, double e_
         if (rms_theta < 0.2) {
             return rms_theta; 
         } else {
-            double nu_g = get_nu_g(e_i, n, k_4);
+            double nu_g = 0.5;
+            if (e_i > 0.7) {    // limit of parametrization
+                nu_g = get_nu_g(e_i, n, k_4);
+            };
+
             auto k_5 = k_4 * pow(nu_g, 1 + n) * pow(1 - nu_g, 0.5 - n);
             rms_theta = k_5 * pow(1 - nu, -0.5);
             if (rms_theta < 0.2) {
                 // If no case matches
-                Logging::Get("proposal.deflection")->warn("BremsGinneken deflection: rms_theta = {} > 0.2, nu_g = {}", rms_theta, nu_g);
+                Logging::Get("proposal.deflection")->warn("BremsGinneken deflection: rms_theta = {} < 0.2, nu_g = {}", rms_theta, nu_g);
```

Comment:
What exactly does this warning mean? Is it problematic if we run into this case?

Reply:
In general, this case should not appear at all. I have just added it as a cross check.

Reply:
In this case, shouldn't we rather throw an exception here, to avoid that the program continues in an undefined state?

Reply:
Now there is an exception instead of a warning